au/BUILD.bazel

```diff
@@ -589,20 +566,19 @@ cc_test(
 cc_library(
     name = "stdx",
     srcs = glob([
-        "code/au/stdx/*.cc",
-        "code/au/stdx/experimental/*.cc",
+        "stdx/*.cc",
+        "stdx/experimental/*.cc",
```

Comment:
lol... Why does this exist?  Looks like I added it in #8.  Really weird.

I'll remove it in a follow on PR.